test/e2e/test.js

```diff
@@ -0,0 +1,226 @@
+const { expect } = require('chai');
+const startInsightUI = require('../../lib/test/startInsightUI');
+const wait = require('../../lib/test/util/wait');
+
+const topPanel = require('../../lib/test/pages/TopPanel');
+const blockPage = require('../../lib/test/pages/BlockPage');
+const statusPage = require('../../lib/test/pages/StatusPage');
+
+const InsightUIOptions = require('../../lib/test/service-ctl/InsightUIOptions');
+
+let originalTimeout;
+
+describe('basic UI tests', () => {
+  let masterNode;
+
+  let url;
+  let blockHash;
+  let trxHash;
+
+  beforeAll(async () => {
+    const rootPath = process.cwd();
+
+    const insightUIContainerOptions = {
+      volumes: [
+        `${rootPath}/dashcore-node:/insight/node_modules/@dashevo/insight-ui/dashcore-node`,
+        `${rootPath}/po:/insight/node_modules/@dashevo/insight-ui/po`,
+        `${rootPath}/public:/insight/node_modules/@dashevo/insight-ui/public`,
+      ],
+    };
+
+    const insighUIOptions = {
+      cacheNodeModules: true,
```

Comment:
This will not work with this shitty structure when your project in the Docker image is a dependency of another one. Just remove `cacheNodeModules` and `localAppPath`.

Reply:
but it works. changes in public & dashcore-node are applied on startup

Reply:
This options not about mounting code, it's about installing new dependencies what you can add in your project but they are absent in Docker image.